api/scripts/holiday_closure_test.ts

```diff
@@ -0,0 +1,104 @@
+// Before the end-of-year closure, we test that the date picker dropdown in the Request item dialog is going to display the correct options
+// to ensure that users can only requests items when they can be available, on a day the library is open
+// Once the Modified opening times have been added to Prismic and published,
+// set dateNow in utils.ts as the date to be tested and run this script with
+// yarn check_holiday_closures
+
+import { DateTime } from 'luxon';
+import { getElasticClient } from '@weco/content-common/services/elasticsearch';
+import { ElasticsearchVenue, Venue } from '@weco/content-common/types/venue';
+import { getNextOpeningDates } from '@weco/content-api/src/controllers/utils';
+import { NextOpeningDate } from '@weco/content-common/types/venue';
+
+// set this to the date to be tested
+const dateNow = new Date();
+
+const formatDate = (openingDay: NextOpeningDate) =>
+  openingDay.open
+    ? new Date(openingDay.open).toUTCString().slice(0, 11) + '\n'
+    : '';
+
+const compareDates = (library: NextOpeningDate, deepstore: NextOpeningDate) => {
+  const libraryDate =
+    library.open && DateTime.fromJSDate(new Date(library.open)).startOf('day');
+  const deepstoreDate =
+    deepstore.open &&
+    DateTime.fromJSDate(new Date(deepstore.open)).startOf('day');
+  if (libraryDate === undefined || deepstoreDate === undefined) {
+    throw 'One of these dates is undefined!';
```

Comment:
I'm getting an eslint error here (it should've been caught automatically but I guess there's still improvements to do on this repo!)

> Expected an error object to be thrown.
> https://eslint.org/docs/latest/rules/no-throw-literal